build.gradle.kts

```diff
@@ -14,6 +14,10 @@ buildscript {
         // in the individual module build.gradle files
     }
 }
+plugins {
+  id("de.fayard.buildSrcVersions") version "0.4.2"
```

Comment:
Please also indent using 4 spaces, like the rest of this and other Gradle files.